src/SFA.DAS.Forecasting.Api/Models/ApprenticeshipCourse.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using SFA.DAS.Forecasting.InnerApi.Responses;
+
+namespace SFA.DAS.Forecasting.Api.Models
+{
+    public class ApprenticeshipCourse
+    {
+        public string Id { get; set; }
+        public string Title { get; set; }
+        public decimal FundingCap { get; set; }
+        public int Level { get; set; }
+        public int Duration { get; set; }
+        public ApprenticeshipCourseType CourseType { get; set; }
+        public List<FundingPeriod> FundingPeriods { get; set; }
+
+        public static implicit operator ApprenticeshipCourse(GetStandardsListItem standard)
+        {
+            return new ApprenticeshipCourse
+            {
+                Id = standard.Id,
+                Title = standard.Title,
+                FundingCap = standard.FundingCap,
+                Level = standard.Level,
+                Duration = standard.ApprenticeshipFunding?.FirstOrDefault()?.Duration ?? 0,
```

Comment:
The duration logic needs to be the same as what is being done for finding the funding cap, rather than the first one